UIPS/UIP-0101.md

```diff
@@ -13,6 +13,9 @@ created: 2023-06-29
 
 This proposal introduces a new vane, `%lick`, designed to function as an Interprocess Communication (IPC) vane. It enables communication between Urbit applications and applications operating on the Host Operating System (Host OS). Urbit applications, through this vane, can open and close IPC ports on the Host OS, receive notifications about POSIX applications' connectivity status, and transfer jammed nouns.
 
+## Status
+`%lick` was released in 412k.
+
```

Comment:
If you're going to add this to documents whose status is otherwise unchanged, I would link to relevant PRs and/or releases.

Reply:
Removed a bunch of these